modules/ocf_desktop/manifests/packages.pp

```diff
@@ -62,7 +62,8 @@
   # rstudio updates to libgstreamer1.0-0 and libgstreamer-plugins-base1.0-0.
   package { 'rstudio':; }
 
-  # TODO: install remmina from backports when that becomes available
+  # Install remmina from backports.
+  package { 'remmina':; }
```

Comment:
Nice, that should work, I'd probably include it in the list of packages above instead of keeping it separate (because although it's coming from backports, that's not especially important), but looks good.